readthedocs_theme/templates/includes/topnav.html

```diff
@@ -60,11 +60,12 @@
       <code>https://readthedocs.com</code>
     </p>
   </a>
-  <div class="ui divider"></div>
-  <a class="item" href="{{ SITE_URL }}/about/">
+  <!-- Hide the learn more, we have a lot of Learn More links.. -->
+  <!-- <div class="ui divider"></div>
+  <a class="item" href="{{ SITE_URL }}/features/">
```

Comment:
Also seems good to just remove this one, but if not:

```suggestion
  {# Hide the learn more, we have a lot of Learn More links.. #}
  {# <div class="ui divider"></div>
  <a class="item" href="{{ SITE_URL }}/features/">
```